interface-definitions/interfaces-openvpn.xml.in

```diff
@@ -635,6 +635,87 @@
                 </properties>
                 <defaultValue>net30</defaultValue>
               </leafNode>
+              <node name="2fa">
+                <properties>
+                  <help>2-factor authentication</help>
+                </properties>
+                <children>
+                  <node name="totp">
+                    <properties>
+                      <help>Time-based One-Time Passwords</help>
+                    </properties>
+                    <children>
+                      <leafNode name="slop">
+                        <properties>
+                          <help>Maximum allowed clock slop in seconds (default: 180)</help>
+                          <valueHelp>
+                            <format>1-65535</format>
+                            <description>Seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-65535"/>
+                          </constraint>
+                        </properties>
+                      </leafNode>
+                      <leafNode name="drift">
+                        <properties>
+                          <help>time drift in seconds (default: 0)</help>
+                          <valueHelp>
+                            <format>1-65535</format>
+                            <description>Seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-65535"/>
+                          </constraint>
+                        </properties>
+                      </leafNode>
+                      <leafNode name="step">
+                        <properties>
+                          <help>Step value for TOTP in seconds (default: 30)</help>
+                          <valueHelp>
+                            <format>1-65535</format>
+                            <description>Seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-65535"/>
+                          </constraint>
+                        </properties>
+                      </leafNode>
+                      <leafNode name="digits">
+                        <properties>
+                          <help>Number of digits to use from TOTP hash (default: 6)</help>
+                          <valueHelp>
+                            <format>1-65535</format>
+                            <description>Seconds</description>
```

Comment:
Should we maybe use lowercase in descriptions (since those are used in the inline help), not sure.